src/site/content/en/blog/user-preference-media-features-headers/index.md

```diff
@@ -0,0 +1,186 @@
+| `Sec-CH-Prefers-Reduced-Data`         | [`no-preference`](https://drafts.csswg.org/mediaqueries-5/#valdef-media-prefers-reduced-data-no-preference), [`reduce`](https://drafts.csswg.org/mediaqueries-5/#valdef-media-prefers-reduced-data-reduce)                                                                                                                                                                       | [`prefers-reduced-data`](https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-data)                 |
+
+## Browser support
+
+The `Sec-CH-Prefers-Color-Scheme` client hint header is supported on Chromium&nbsp;93.
+Other vendors' feedback, namely [WebKit's](https://lists.webkit.org/pipermail/webkit-dev/2021-May/031856.html)
+and [Mozilla's](https://github.com/mozilla/standards-positions/issues/526), is pending.
+
+## Demo of `Sec-CH-Prefers-Color-Scheme`
+
+Try the [demo](https://sec-ch-prefers-color-scheme.glitch.me/) in Chromium&nbsp;93 and notice how
+the inlined CSS changes according to the user's preferred color scheme.
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/pKAKyrN18CjhAYUNpJyk.png", alt="Sec-CH-Prefers-Color-Scheme: dark", width="800", height="541" %}
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/6Xujcgyveo0QY0E3LQOF.png", alt="Sec-CH-Prefers-Color-Scheme: light", width="800", height="541" %}
+
+## How it works
+
+1. The client makes an initial request to the server.
+   ```bash
+   GET / HTTP/2
+   Host: example.com
+   ```
+1. The server responds, telling the client via `Accept-CH` that it accepts the
+   `Sec-CH-Prefers-Color-Scheme` and the `Sec-CH-Prefers-Contrast` Client Hints, out of which as per
+   `Critical-CH` it considers `Sec-CH-Prefers-Color-Scheme` a Critical Client Hint that it also
+   varies the response on as conveyed by `Vary`.
+   ```bash
+   HTTP/1.1 200 OK
+   Content-Type: text/html
+   Accept-CH: Sec-CH-Prefers-Color-Scheme, Sec-CH-Prefers-Contrast
+   Vary: Sec-CH-Prefers-Color-Scheme
+   Critical-CH: Sec-CH-Prefers-Color-Scheme
+   ```
+1. The client then retries the request, telling the server via `Sec-CH-Prefers-Color-Scheme` that it
+   has a user preference for dark-schemed content.
+   ```bash
+   GET / HTTP/2
+   Host: example.com
+   Sec-CH-Prefers-Color-Scheme: "dark"
+   ```
+1. The server can then tailor the response to the client's preferences accordingly and, for example,
+   inline the CSS responsible for the dark theme into the response body.
+
+## Privacy and Security Considerations
```

Comment:
```suggestion
## Node.js example

The Node.js example below, written for the popular Express.js framework, shows how
dealing with the `Sec-CH-Prefers-Color-Scheme` client hint header could look like in practice.
This code is what actually powers the [demo](#demo-of-sec-ch-prefers-color-scheme) above.

```js
app.get("/", (req, res) => {
  // Tell the client the server accepts the `Sec-CH-Prefers-Color-Scheme` client hint…
  res.set("Accept-CH", "Sec-CH-Prefers-Color-Scheme");
  // …and that the server's response will vary based on its value…
  res.set("Vary", "Sec-CH-Prefers-Color-Scheme");
  // …and that the server considers this client hint a _critical_ client hint.
  res.set("Critical-CH", "Sec-CH-Prefers-Color-Scheme");
  // Read the user's preferred color scheme from the headers…
  const prefersColorScheme = req.get("sec-ch-prefers-color-scheme");
  // …and send the adequate HTML response with the right CSS inlined.
  res.send(getHTML(prefersColorScheme));
});
```

## Privacy and Security Considerations
```